network-config-analyzer/DimensionsManager.py

```diff
@@ -28,10 +29,13 @@ def __init__(self):
             self.domain_str_to_dfa_map = dict()
             dfa_all_words_default = self._get_dfa_from_alphabet_str(self.default_dfa_alphabet_str)
             ports_interval = CanonicalIntervalSet.get_interval_set(1, 65535)
+            all_methods_interval = MethodSet.whole_range_interval_set()
+            dfa_all_methods = MinDFA.dfa_from_regex(MethodSet.all_methods_regex())
             self.dim_dict = dict()
             self.dim_dict["src_ports"] = (DimensionsManager.DimensionType.IntervalSet, ports_interval)
             self.dim_dict["dst_ports"] = (DimensionsManager.DimensionType.IntervalSet, ports_interval)
-            self.dim_dict["methods"] = (DimensionsManager.DimensionType.DFA, dfa_all_words_default)
+            self.dim_dict["methods_dfa"] = (DimensionsManager.DimensionType.DFA, dfa_all_methods)
```

Comment:
we should only have dimension "methods" of type IntervalSet, and remove the dimension "methods_dfa"